drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -584,12 +585,12 @@ void RasterizerCanvasGLES3::_canvas_item_render_commands(Item *p_item, Item *cur
 					}
 
 					if (rect->flags & CANVAS_RECT_TRANSPOSE) {
-						//err..
+						dst_rect.size.x *= -1; // Enconding in the dst_rect.z uniform
```

Comment:
I guess `codespell` will fix that one ;)

Reply:
You don't like the way I spelted it? ;)